src/GrumPHP/Console/Helper/PathsHelper.php

```diff
@@ -80,9 +80,16 @@ public function getAsciiPath()
      */
     public function getAsciiContent($resource)
     {
-        $file = $this->getAsciiPath() . $resource . '.txt';
-        if (!$this->fileSystem->exists($file)) {
-            throw new RuntimeException(sprintf('ASCII file %s could not be found.', $resource));
```

Comment:
I think it is better to continue script execution even if there is no ascii image